distro/ddl/src/main/db-setup/ReadMe.txt

```diff
@@ -0,0 +1,38 @@
+This directory contains setup instructions and scripts for required live databases
```

Comment:
Yes, will do.  I've already done that on my side.  Will add an .md in the project root.